tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,5 @@
+function convertToF(celsius) {
```

Comment:
same as above

Reply:
what is above?

Reply:
Oh, i deleted this comment, nevermind.

Reply:
@agnieszka-pytel all ok here, i just forgot to delete it. 👍